executors/rust/src/mod.rs

```diff
@@ -0,0 +1,22 @@
+mod compat;
+mod try_or_return_error;
+
+pub mod collator;
+pub mod decimalfmt;
+pub mod displaynames;
+pub mod likelysubtags;
+pub mod listfmt;
+pub mod localenames;
+pub mod numberfmt;
+pub mod pluralrules;
+pub mod relativedatetime_fmt;
+
+#[cfg(any(ver = "1.3", ver = "1.4", ver = "1.5"))]
+#[path = "datetime_1.rs"]
```

Comment:
Interesting. I like the way you did this. I didn't know that you can configure which file Cargo should look at to find a particular module, but that's really convenient for this use case.